Ahora Carlos va a ser uso de su código. Acaba de implementar `cantidad_de_positivos` que recibe una colección de números y devuelve la cantidad de positivos.

Nosotros queremos probar la siguiente colección: `[2,4,0,-1]` (la cual debería devolver 2).

Hasta ahora implementamos un simple `assert` que valida que algo sea cierto. Eso es útil, pero a veces el mensaje de error es demasiado genérico y podríamos hacer algo mejor.

> Implementá un ```assert_equals(valor_esperado, valor_obtenido)``` que falle cuando los valores sean distintos y devuelva un mensaje expresivo al respecto.

> Ah... y hacé que ```assert_equals``` reuse ```assert```

